apps/dash-medical-provider-charges/app.py

```diff
@@ -0,0 +1,736 @@
+import dash
+import dash_table
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.graph_objs as go
+from dash.dependencies import State, Input, Output
+from dash.exceptions import PreventUpdate
+
+import pandas as pd
+import os
+
+app = dash.Dash(__name__)
+server = app.server
+
+app.config["suppress_callback_exceptions"] = True
+
+# Plotly mapbox token
+mapbox_access_token = "pk.eyJ1IjoicGxvdGx5bWFwYm94IiwiYSI6ImNqdnBvNDMyaTAxYzkzeW5ubWdpZ2VjbmMifQ.TXcBE-xg9BFdV2ocecc_7g"
+
+state_list = [
+    "AL",
+    "AZ",
+    "AR",
+    "CA",
+    "CO",
+    "CT",
+    "DC",
+    "FL",
+    "GA",
+    "IL",
+    "IN",
+    "IA",
+    "KY",
+    "MD",
+    "MA",
+    "MI",
+    "MN",
+    "MO",
+    "NE",
+    "NJ",
+    "NY",
+    "NC",
+    "OH",
+    "OK",
+    "OR",
+    "PA",
+    "SC",
+    "TN",
+    "TX",
+    "UT",
+    "VA",
+    "WA",
+    "WI",
+    "AK",
+    "DE",
+    "HI",
+    "ID",
+    "KS",
+    "LA",
+    "ME",
+    "MS",
+    "MT",
+    "NV",
+    "NH",
+    "NM",
+    "ND",
+    "RI",
+    "SD",
+    "WV",
+    "VT",
+    "WY",
+]
+
+state_map = {
```

Comment:
I don't think you need both a list and a dictionary, if all of the keys are the same. The list can be extracted from the dictionary -- `state_list` is the same as `state_map.keys()`.

Reply:
💯